lib/atp_scraper/activity.rb

```diff
@@ -46,9 +46,9 @@ def create_record(record, tournament)
       {
         year: tournament[:year],
         player_name: @player_name,
-        player_rank: tournament[:ranking],
+        player_rank: convert_ranking(tournament[:ranking]),
```

Comment:
__[reek]__
* [FeatureEnvy] refers to tournament more than self (maybe move it to another class?) [:ref](https://github.com/troessner/reek/blob/master/docs/Feature-Envy.md)